codalab/lib/download_manager.py

```diff
@@ -288,7 +288,8 @@ def _get_read_response_stream(self, response_socket_id):
 
     def _get_read_response_string(self, response_socket_id):
```

Comment:
Should we rename this `_get_read_response_bytestring` to avoid confusion?

Reply:
I'm not sure what "bytestring" would mean in python 3, so I think I'll just rename it to `_get_read_response` so that the name isn't misleading (calling it `_get_read_response_bytes` sounds superfluous because then we should rename all our functions to have the return type appended to its name).